application/libraries/Api/Rest/V1/SchemaFactory/SchemaFactorySurvey.php

```diff
@@ -3,14 +3,18 @@
 namespace LimeSurvey\Api\Rest\V1\SchemaFactory;
 
 use GoldSpecDigital\ObjectOrientedOAS\Objects\Schema;
+use GoldSpecDigital\ObjectOrientedOAS\Contracts\SchemaContract;
 
 class SchemaFactorySurvey
 {
     /**
-     *  @SuppressWarnings(PHPMD.ExcessiveMethodLength)
+     * @SuppressWarnings(PHPMD.ExcessiveMethodLength)
+     * @param ?\GoldSpecDigital\ObjectOrientedOAS\Contracts\SchemaContract[] $properties
      */
-    public function make(): Schema
+    public function make(SchemaContract ...$properties): Schema
     {
+        $properties = !empty($properties) ? $properties : [];
```

Comment:
The fact that we have to change multiple places in the code in the same way is kinda a tell, no? :) Fine for now tho.

Reply:
Do you have a suggestion for a better design here?

Reply:
Hm in this particular case, I guess remove the nullable type, so that `$properties` cannot be null? Would that work?

Reply:
No. Do you want to factor out the null coalescing operator?

Reply:
The $properties argument is nullable because its optional. Its not always needed.

Reply:
Ah and you can't combine `...` with `= []` ?

Reply:
Seems not. I got error "Variadic parameter cannot have a default value".